packages/edit-post/src/components/layout/index.native.js

```diff
@@ -151,6 +154,7 @@ class Layout extends Component {
 						) }
 						<Header />
 						<BottomSheetSettings />
+						<FloatingToolbar.Slot />
```

Comment:
It seems moving `FloatingToolbar.Slot` here make it unfocusable at all on Android (see [this comment](https://github.com/WordPress/gutenberg/pull/21446#issuecomment-614136356))